packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -223,29 +235,10 @@ export function BindStateToController(
 export type I18nState = Record<string, (variables?: TOptions) => string>;
 
 /**
- * Decorator to be used on a property decorator with Stencil's `State` that will be subscribed automatically to the i18next language change.
- * The state should be of the `I18nState` format and use the `i18n` binding to retrieve strings.
+ * TODO: remove this method once no component calls it
+ * @deprecated This binding method is not needed anymore and should be removed.
  */
 export function BindStateToI18n() {
-  return (component: InitializableComponent, stateProperty: string) => {
-    const {disconnectedCallback, initialize} = component;
-    let unsubscribe = () => {};
-
-    component.initialize = function () {
-      const updateStrings = () => {
-        this[stateProperty] = {...this[stateProperty]};
-      };
-      updateStrings(); // Ensures re-render of localized strings on initialization
-      this.bindings.i18n.on('languageChanged', updateStrings);
-      unsubscribe = () =>
-        this.bindings.i18n.off('languageChanged', updateStrings);
-
-      initialize && initialize.call(this);
-    };
-
-    component.disconnectedCallback = function () {
-      unsubscribe();
-      disconnectedCallback && disconnectedCallback.call(this);
-    };
-  };
```

Comment:
Could the `forceUpdate` step have been added in here instead of moving the code to `InitializeBindings`? It seems like a better separation of concerns, and makes it possible to force update only the components that have localized strings.

Reply:
https://github.com/coveo/ui-kit/pull/861/files#r644972487
To that I would add that the boilerplate removal is a gain